yarn-project/p2p/src/client/p2p_client.test.ts

```diff
@@ -194,7 +194,7 @@ describe('In-Memory P2P Client', () => {
     ];
 
     for (const quote of proofQuotes) {
-      client.broadcastEpochProofQuote(quote);
+      await client.addEpochProofQuote(quote);
```

Comment:
Stack the layers. 

![Pancake Day Dancing GIF by Aya Murata](https://media1.giphy.com/media/ibLuBzLCXEJ6ipgLqx/giphy.gif)